components/automate-cli/pkg/verifyserver/utils/db/db.go

```diff
@@ -0,0 +1,30 @@
+package db
```

Comment:
Please format this file again, its not indented properly

Reply:
done